native_client/README.md

```diff
@@ -60,28 +71,26 @@ If you'd like to build the language bindings or the decoder package, you'll also
 * [SWIG](http://www.swig.org/)
 * [node-pre-gyp](https://github.com/mapbox/node-pre-gyp) (for Node.JS bindings only)
 
-## Preparation
 
-Create a symbolic link in your TensorFlow checkout to the DeepSpeech `native_client` directory. If your DeepSpeech and TensorFlow checkouts are side by side in the same directory, do:
+### Building your own Binaries
+
+
+Firstly, you should create a symbolic link in your TensorFlow checkout to the DeepSpeech `native_client` directory. If your DeepSpeech and TensorFlow checkouts are side by side in the same directory, do:
 
 ```
 cd tensorflow
 ln -s ../DeepSpeech/native_client ./
 ```
 
-## Building
-
-Before building the DeepSpeech client libraries, you will need to prepare your environment to configure and build TensorFlow. 
-Preferably, checkout the version of tensorflow which is currently supported by DeepSpeech (see requirements.txt), and use the bazel version recommended by TensorFlow for that version.
-Then, follow the [instructions](https://www.tensorflow.org/install/install_sources) on the TensorFlow site for your platform, up to the end of 'Configure the installation'.
+Next, you will need to prepare your environment to configure and build TensorFlow. Preferably, checkout the version of `tensorflow` which is currently supported by DeepSpeech (see requirements.txt), and use the `bazel` version recommended by TensorFlow for that version. Follow the [instructions](https://www.tensorflow.org/install/install_sources) on the TensorFlow site for your platform, up to the end of ["Configure the Build"](https://www.tensorflow.org/install/source#configure_the_build).
```

Comment:
I think this can potentially trap people: you have to clone our fork of TensorFlow, not the upstream repo, as the "Building & Installing your own Binaries" section says. Upstream TensorFlow will not build without modifications to `native_cient/BUILD`.

Reply:
@reuben - what do you suggest we say here instead?

Reply:
We should link to https://github.com/mozilla/tensorflow explicitly.

"Clone https://github.com/mozilla/tensorflow, then preferably checkout the version which is currently..."

Reply:
see 14016b818818a1786fb0b76574fc48f8c283a47a